### DISCUSSION TITLE
customised bar Toggler

### DISCUSSION THREAD

**Discussion by salucien** [Megabit/Blazorise#2386] [Q&A] [answered]
Hi !
I'm trying to create a custom bar Toggler with an svg, something like this : 

```
<BarToggler >
        @customToggle
</BarToggler>
<BarBrand>
    <BarItem Class="nav-item">
        <BarLink To="dashboard">
            Blazorise
        </BarLink>
    </BarItem>
</BarBrand>
```

But it's working as a link when i click on it and go to the Link from my Barlink: 

![image](https://user-images.githubusercontent.com/76843827/118954827-6fe0cd00-b95e-11eb-813b-752b8852868f.png)

What am I doing wrong ?

**Comment by salucien** [comment, marked as answer]
Nevermind I found it...
```
<BarBrand>
    <BarItem Class="nav-item">
        <BarLink To="dashboard">
            Blazorise
        </BarLink>
    </BarItem>
</BarBrand>
<BarToggler ChildContent="@customToggle" />
```

**Comment by stsrki** [comment]
You beat me by seconds. I was just writing the response :)

**Reply by salucien** [reply]
sorry for the inconvenience :)
